src/main/java/PersonalBudget/business/income/domain/repository/IncomeCategoryRepository.java

```diff
@@ -0,0 +1,34 @@
+package PersonalBudget.business.income.domain.repository;
+
+import PersonalBudget.business.income.domain.model.IncomeCategoryEntity;
+import jakarta.transaction.Transactional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface IncomeCategoryRepository extends JpaRepository<IncomeCategoryEntity, Long> {
+
+    @Modifying
+    @Transactional
+    @Query(value = "insert into incomes_category_assigned_to_users (user_id, name) " +
+            "values (:userId, 'Salary'), " +
+            "(:userId, 'Interest'), " +
+            "(:userId, 'Allegro'), " +
+            "(:userId, 'Another')",
+            nativeQuery = true)
```

Comment:
The default values shouldn't be hardcoded in the query, let me think how we can make it better